### PR TITLE
Added 6 tests

### PR DESCRIPTION
**Pull request by gantz26**
Created automated tests for C200, C201, C202, C207, C576, C577

### CODE REVIEW

File: tests/login.spec.ts [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        mainPage = new MainPage(page);
+        loginPage = new LoginPage(page);
+        profilePage = new ProfilePage(page);
+        await mainPage.open();
+    });
+
+    test("Authorization with empty fields", async ({ page }) => {
+        await loginPage.clickEnterButton();
+        await loginPage.clickLoginButton();
+        await page.waitForLoadState("networkidle");
```
Comment (joh-kepler) [inline tests/login.spec.ts]: Using waitForLoadState("networkidle") is a good practice, but consider using specific waits for elements to appear or be hidden when possible to make tests more robust against network delays.

File: tests/login.spec.ts [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+            "991234785",
+            "099123478",
+            "+380-99-123-4785",
+            "+380 99 123 4785",
+            "+380(99)1234785",
+            "(99)1234785",
+            "09912347850",
+            "+100991234785",
+            "+0991234785"
+        ];
```
Comment (joh-kepler) [inline tests/login.spec.ts]: Here and further, instead of using hardcoded data, you can use some randomizer to avoid the Pesticide Paradox loop.

**Review by joh-kepler — COMMENTED** [review]
Hi, overall, the test is quite comprehensive and well-structured. I have nothing to add more, except using const instead of random in checks.

File: tests/login.spec.ts (hunk 2 above)
Comment (gantz26) [reply tests/login.spec.ts]: The hardcoded data are designed to represent specific patterns and edge cases (e.g., missing digits, incorrect country codes, extra characters) for the specific account. When dealing with invalid cases, it's crucial to know exactly what format caused the validation to fail. Randomized data might lead to inconsistent results or missed validations over time.

Comment (joh-kepler) [reply tests/login.spec.ts]: You can configure the randomizer function to pull the data construction you want, just to avoid repeating. This may be a non-trivial way, but it's definitely the best.

Comment (gantz26) [reply tests/login.spec.ts]: It can work for phone numbers, but how about the email and the password? They all are mostly the same, only incorrect symbols are added to them

**Review by joh-kepler — APPROVED** [review]
